### PR TITLE
Automated unit testing

### PR DESCRIPTION
**Pull request by swift-kim**
- Remove headed engine tests, because they cannot be run without a display attached.
- Add the `test` job to the CI pipelines.

### CODE REVIEW

File: shell/platform/tizen/flutter_tizen_engine_unittest.cc [hunk 1]
```diff
@@ -79,11 +71,6 @@ TEST_F(FlutterTizenEngineTest, GetTextureRegistrar) {
   EXPECT_TRUE(engine_->GetTextureRegistrar() == nullptr);
 }
 
-TEST_F(FlutterTizenEngineTestHeaded, GetTextureRegistrar) {
```
Comment (bwikbs) [inline shell/platform/tizen/flutter_tizen_engine_unittest.cc:82]: How about just disable it instead of removing it?

File: azure-pipelines.yml [hunk 2]
```diff
@@ -100,8 +100,29 @@ jobs:
     failOnStderr: true
   - publish: $(Build.StagingDirectory)
     artifact: $(System.JobName)
-- job: release
+- job: test
```
Comment (bwikbs) [inline azure-pipelines.yml:103]: I am going to do this when a github action is enable, but you do it right away! :+1: :+1:

Comment (swift-kim) [reply azure-pipelines.yml:103]: It was as simple as installing EFL on the CI server and adding 20 lines of code. :)

**Review by bwikbs — APPROVED** [review]
It's pretty cool! thanks!

**Comment by swift-kim** [comment]
I don't know why but apparently the incremental builds take longer than before (1 min → 3-4 min).

**Pull request merged**